homeassistant/components/contact_energy_nz/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "contact_energy_nz",
+  "name": "Contact Energy NZ",
+  "codeowners": ["@tkhadimullin"],
+  "config_flow": true,
+  "dependencies": [],
+  "documentation": "https://www.home-assistant.io/integrations/contact_energy_nz",
+  "integration_type": "service",
+  "iot_class": "cloud_polling",
+  "loggers": ["contact_energy_nz"],
+  "requirements": ["contact-energy-nz==0.1.4684953769"]
```

Comment:
Can you please add a link to this library's source to the description?

Please also update the information on pypi.org to include a link to the source.